docs/indepth/installation.rst

```diff
@@ -379,6 +379,58 @@ Your firewall will need to allow access to the ports your applications are expos
    Keep in mind the consequences of exposing unsecured services to the Internet.
    Both applications with exposed ports and applications accessed via links will be accessible by anyone on the Internet.
 
+ZFS Backend Configuration
+-------------------------
+
+The ZFS backend requires ZFS to be installed.
+
+
+Installing ZFS on CentOS 7
+..........................
+
+Installing ZFS requires the kernel development headers for the running kernel.
+Since CentOS doesn't provide easy access to old package versions,
+the easiest way to get appropriate headers is to upgrade the kernel and install the headers.
```

Comment:
:question: This implies that there is at least one another way which is not as easy. I may have my own reasons for doing it that other way (whether something is "easy" is a combination of whether it is simple or complex and a person's experience :smile: ). If I do this any other way is it supported by Flocker? If not, perhaps make it more clear that this is **the** way to do things.